app/controllers/dashboard/cards_controller.rb

```diff
@@ -35,6 +37,12 @@ def destroy
     respond_with @card
   end
 
+  def get_flickr_images
+    @flickr_images = FlickrService.search_photos(params[:search],
+                                                 '1, 2, 3, 4, 5, 6, 7',
+                                                 'url_sq, url_m', 10, 1)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.